android/.settings/org.eclipse.buildship.core.prefs

```diff
@@ -1,11 +1,11 @@
-arguments=
+arguments=--init-script /var/folders/ck/r25tcygs77n6hv8d515p1w_c0000gn/T/d146c9752a26f79b52047fb6dc6ed385d064e120494f96f08ca63a317c41f94c.gradle --init-script /var/folders/ck/r25tcygs77n6hv8d515p1w_c0000gn/T/52cde0cfcf3e28b8b7510e992210d9614505e0911af0c190bd590d7158574963.gradle
```

Comment:
1) 
What

Reply:
I wish I knew. React-native just did this when I installed Android. Thanks, I hate it.

Reply:
I'm just going to revert it lol